### ISSUE TITLE
rplot rearrange 

### ISSUE THREAD

**Issue by xmarti6** [tidymodels/corrr#79]
Hi, "rplot" do not respect rearrange order (corrr(v0.4.0)), no matter what method.
```
rplot(rearrange(x, method = "MDS", absolute = F))+ theme(axis.text.x = element_text(angle = 90, hjust = 1))
rplot(rearrange(x, method = "HC", absolute = F))+ theme(axis.text.x = element_text(angle = 90, hjust = 1))
```
produce same output. "%>% rearrange(...)" pipe variant not working either.

Thank you

**Comment by topepo** [comment]
Can you please provide a minimal reprex (reproducible example)? The goal of a reprex is to make it as easy as possible for me to recreate your problem so that I can fix it: please help me help you!

If you've never heard of a reprex before, start by reading "[What is a reprex](https://github.com/tidyverse/reprex#what-is-a-reprex)", and follow the advice further down that page.

**Comment by juliasilge** [comment]
Closed by #99
